charts/wikijs/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 apiVersion: v2
 appVersion: 2.5.170
-version: 0.1.11
+version: 0.2.1
```

Comment:
Please change this to `0.2.0` since it's the first release with an added feature.

Reply:
Please also update line [28](https://github.com/nicholaswilde/helm-charts/blob/2a03ac3f33d67eee628823c17969405acf731185/charts/wikijs/Chart.yaml#L28) with the reason for change.